livestyled/resource_client.py

```diff
@@ -963,6 +980,20 @@ def create_product_variant_stock(
     ) -> ProductVariantStock:
         return self._create_resource(ProductVariantStockSchema, product_variant_stock)
 
+    def get_product_variant_stocks(
+            self,
+            composite_id: str
```

Comment:
Do you think we should have two parameters? `fulfilment_point` and `product_variant` and allow the method do the joining? With this approach, the calling lambda needs to do this. 

So all we will need is `get_product_variant_tocks(self, fulfilment_point: int, product_variant_id: int):...`

And make the request within accordingly